src/main/java/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import java.io.IOException;
+
+public interface FileService {
+    String readFromFile(String filePath) throws IOException;
+
+    void writeToFile(String filePath, String report) throws IOException;
```

Comment:
```suggestion
    List<String> readFromFile(String filePath);

    void writeToFile(String filePath, String report);
```
почему String **report**?